bipentropy/bipentropy.py

```diff
@@ -38,26 +50,35 @@ def bip32_xprv_to_entropy(self, path, xprv_string):
             raise ValueError('ERROR: Invalid xprv')
         return self.__hmac_sha512(self.__derive_k(path, xprv))
 
-    def entropy_from_wif(self, wif):
-        return self.__hmac_sha512(self.__get_k_from_wif(wif))
+    def bip32_xprv_to_hex(self, index, width, xprv_string):
+        # export entropy as hex
+        path = f"83696968p/128169p/{width}p/{index}p"
```

Comment:
I don't agree we should fix the paths here. While you can use those paths, you may not want to, in the same was people may chose not to use BIP44 derivation paths for their wallets. This is the wrong place to enforce the path, as the library is simply performing the transformation of entropy to end result. If you want to standardize it it would need to be in a wrapper above (but I think this is not desirable).